packages/core/app/fs-next-update/page.tsx

```diff
@@ -1,3 +1,63 @@
-export default function Page() {
-  return <div>placeholder</div>
+import type { PageContentType } from 'src/server/cms'
+import { getPage } from 'src/server/cms'
+import CUSTOM_COMPONENTS from 'src/customizations/src/components'
+
+import { getDynamicContent } from 'src/utils/dynamicContent'
+import storeConfig from '../../faststore.config'
+import { ComponentType } from 'react'
+import { OverriddenDefaultHero as Hero } from 'src/components/sections/Hero/OverriddenDefaultHero'
+import PageProvider from 'src/sdk/overrides/PageProvider'
+import RenderSections from 'app/components/cms/RenderSections'
```

Comment:
**suggestion (non-blocking):** team, how about we leverage this migration to start reordering the import list? I particularly prefer when the framework imports come first.

Reply:
I think we should use an eslint-plugin to order the imports

Reply:
okay! Sounds good to me! :)